src/components/layout/MenuBar/MenuBar.js

```diff
@@ -15,12 +16,12 @@ const MenuBar = ({ children }) => (
         <div className={'col-auto ' + styles.menu}>
           <ul>
             <li>
-              <a href='/#' className={styles.active}>
-                Home
-              </a>
+              <a href='/#'>Home</a>
             </li>
             <li>
-              <a href='/#'>Furniture</a>
+              <Link id={id} exact to={'/shop/' + id} className={styles.active}>
```

Comment:
Co miało by oznaczać to `id`, ten props ma wartość `undefined` Tutaj wsytarczy wpisać nazwe kategori w linku typu `/shop/furniture`